example.go

```diff
@@ -0,0 +1,376 @@
+	createTopicOptionsModel := new(adminrestv1.CreateTopicOptions)
+	createTopicOptionsModel.Name = core.StringPtr("test-topic")
+	createTopicOptionsModel.Partitions = core.Int64Ptr(int64(26))
+	createTopicOptionsModel.PartitionCount = core.Int64Ptr(int64(1))
+
+	// Create the Topic.
+	response, operationErr := serviceAPI.CreateTopic(createTopicOptionsModel)
+	if operationErr != nil {
+		return fmt.Errorf("Error Creating Topics: %s\n", operationErr.Error())
+	}
+
+	// Check the result.
+	if response.StatusCode != http.StatusAccepted {
+		return fmt.Errorf("Error Creating Topic: status %d\n", response.StatusCode)
+	}
+
+	fmt.Printf("\tname: %s created\n", *createTopicOptionsModel.Name)
+
+	return nil
+} // func.end
+
+func deleteTopic(serviceAPI *adminrestv1.AdminrestV1) error {
+	// Construct an instance of the DeleteTopicOptions model
+	deleteTopicOptionsModel := new(adminrestv1.DeleteTopicOptions)
+	deleteTopicOptionsModel.TopicName = core.StringPtr("test-topic")
+	deleteTopicOptionsModel.Headers = map[string]string{"x-custom-header": "x-custom-value"}
+
+	// Delete Topic
+	response, operationErr := serviceAPI.DeleteTopic(deleteTopicOptionsModel)
+	if operationErr != nil {
+		return fmt.Errorf("Error Deleting Topic: %s\n", operationErr.Error())
+	}
+
+	// Check the result.
+	if response.StatusCode != http.StatusAccepted {
+		return fmt.Errorf("Error Deleting Topic: status %d\n", response.StatusCode)
+	}
+
+	fmt.Printf("\tname: %s deleted\n", *deleteTopicOptionsModel.TopicName)
+	return nil
+} // func.end
+
+func updateTopicDetails(serviceAPI *adminrestv1.AdminrestV1) error {
+	// Construct an instance of the UpdateTopicOptions model
+	updateTopicOptionsModel := new(adminrestv1.UpdateTopicOptions)
+	updateTopicOptionsModel.TopicName = core.StringPtr("test-topic")
+	updateTopicOptionsModel.NewTotalPartitionCount = core.Int64Ptr(int64(6))
+	updateTopicOptionsModel.Headers = map[string]string{"x-custom-header": "x-custom-value"}
+
+	// Invoke operation with valid options model (positive test)
+	response, operationErr := serviceAPI.UpdateTopic(updateTopicOptionsModel)
+	if operationErr != nil {
+		return fmt.Errorf("Error Updating Topic: %s\n", operationErr.Error())
+	}
+
+	// Check the result.
+	if response.StatusCode != http.StatusAccepted {
+		return fmt.Errorf("Error Updating Topics: status %d\n", response.StatusCode)
+	}
+
+	fmt.Printf("\tname: %s updated\n", *updateTopicOptionsModel.TopicName)
+
+	return nil
+} // func.end
+
+// nolint
+func replaceMirroringTopicSelection(serviceAPI *adminrestv1.AdminrestV1) error {
+	// Construct an instance of the ReplaceMirroringTopicSelectionOptions model
+	replaceMirroringTopicSelectionOptionsModel := new(adminrestv1.ReplaceMirroringTopicSelectionOptions)
+	replaceMirroringTopicSelectionOptionsModel.Includes = []string{"test-topic"}
+	replaceMirroringTopicSelectionOptionsModel.Headers = map[string]string{"x-custom-header": "x-custom-value"}
+
+	// Enable retries.
+	serviceAPI.EnableRetries(0, 0)
```

Comment:
We don't seem to do this in other methods? Is this needed? How are we enabling retries passing twice `0`?